src/DataViews/DataView.cpp

```diff
@@ -228,12 +228,19 @@ void DataView::OnEnableFrameTrackRequested(const std::vector<int>& selection) {
   metrics_uploader_->SendLogEvent(
       orbit_metrics_uploader::OrbitLogEvent::ORBIT_FRAME_TRACK_ENABLE_CLICKED);
 
+  ORBIT_CHECK(app_->HasCaptureData());
+
   for (int i : selection) {
     const FunctionInfo* function = GetFunctionInfoFromRow(i);
     if (function == nullptr) continue;
     // Functions used as frame tracks must be hooked (selected), otherwise the
     // data to produce the frame track will not be captured.
-    app_->SelectFunction(*function);
+    // TODO(b/228151393) The condition is supposed to prevent "selecting" a function when a capture
+    // is loaded with no connection to a process being established. Perhaps, resolution of the
+    // mentioned but would allow for a better way to check it.
+    if (!app_->HasCaptureData() || app_->IsCaptureConnected(app_->GetCaptureData())) {
```

Comment:
Nit: move it out from the for loop such that we don't need to check it everytime.

Reply:
Never mind. I just saw you update this part.